dotcom-rendering/src/apps/server/articleToHtml.tsx

```diff
@@ -0,0 +1,29 @@
+import { getScriptsFromManifest } from '../../lib/assets';
+import { FEArticleType } from '../../types/frontend';
+import { renderToStringWithEmotion } from '../../web/lib/emotion';
+import { pageTemplate } from './pageTemplate';
+
+export const articleToHtml = (
+	article: FEArticleType,
+): {
+	clientScripts: string[];
+	html: string;
+} => {
+	const { html, extractedCss } = renderToStringWithEmotion(<></>);
+
+	const getScriptArrayFromFile = getScriptsFromManifest({
+		platform: 'apps',
+	});
+
+	const clientScripts = getScriptArrayFromFile('index.js');
```

Comment:
Will we want the commercial bundle here as well, as is done in web?

Reply:
we might do yes, but I'm not sure how we plan to implement ads. It might be that we mimic the current AR implementation initially, where the native layer is responsible for rendering the ads, until we understand how the commercial bundle could support offline ads. What do you think?